### DISCUSSION TITLE
Dropdown with search field

### DISCUSSION THREAD

**Discussion by lukaskonvicka** [vikramlearning/blazorbootstrap#577] [Ideas]
Hello,

it would be nice to have dropdown with search field at the top, so user can search the value he/she is looking for.
I have dropdown with 200 items and its not very comfortable to search for the one you need.

Thank you for your great work
Lukas

**Comment by gvreddy04** [comment]
@lukaskonvicka Thank you for trying BlazorBootstrap. Please create an issue for this. We will add this feature in the future release.

**Reply by lukaskonvicka** [reply]
Hello,
thank you for swift reply. I ceated an issue.
Have a nice day
Lukas
